lib/pure/math.nim

```diff
@@ -69,11 +69,6 @@ when defined(c) or defined(cpp):
 
   proc c_signbit(x: SomeFloat): cint {.importc: "signbit", header: "<math.h>".}
 
-  func c_frexp*(x: cfloat, exponent: var cint): cfloat {.
```

Comment:
too early to remove, check git log, IIRC it was deprecated in 1.5

Reply:
Ok.